.github/styles/Vocab/Pulse/accept.txt

```diff
@@ -0,0 +1,17 @@
+Bitbucket
+CLI
+Codacy
+config
+Ctrl
+GitHub
+hostname
+hotfix
+Jira
+md
+namespace
+OAuth
+onboarding
+PagerDuty
+Pulse
+signup
+tooltip
```

Comment:
Now that the accept list is shorter, I noticed two "suspicious" words (that already came from the codacy/docs repository):

- `md`: from what I can understand now, when I added this "word" I was most probably trying to fix a false positive [on this line](https://github.com/codacy/docs/pull/522/files#diff-2b38db947e7b751925903dde5bd91d6be56b2045f051f1e47109a49f7e965f7fR29) by using the wrong solution. :sweat_smile: We're now correctly excluding the Vale analysis inside the include-markdown `{% ... %}` syntax so this is no longer necessary.
- `signup`: I incorrectly added this word to fix an issue that Vale detected [on this line](https://github.com/codacy/docs/commit/e13ddb167b8c292825d4ae02e3840d36b23c4150#diff-cd48b581f65da59f08e65d645e60f3eacc14f81c1c2a07c958f006dd63366b45R16). But indeed, the "signup" spelling [is incorrect](https://writingexplained.org/sign-up-or-sign-up-difference) and should be "sign-up" instead! :sweat_smile: 

So, it's better to just remove these words from the list. I'm going to do the same thing on codacy/docs. :wink: 

```suggestion
Bitbucket
CLI
Codacy
config
Ctrl
GitHub
hostname
hotfix
Jira
namespace
OAuth
onboarding
PagerDuty
Pulse
tooltip
```

Reply:
Great! I also removed those words from Pulse's list.